### PR TITLE
Quota example

### PR DESCRIPTION
**Pull request by simonqbs**

### CODE REVIEW

File: server/monitoring/monitoring.go [hunk 1]
```diff
@@ -48,6 +51,7 @@ func Monit() {
 		} else {
 			if found > 0 {
 				logger.Info("monitoring", fmt.Sprintf("Deleted %d entries and files in %s", found, time.Since(now)))
+				statistics.S.UpdateCurrentSize(sizeRemoved)
```
Comment (Depado) [inline server/monitoring/monitoring.go]: Are you sure you're not updating to the wrong size ? Might be more something like `statistics.S.UpdateCurrentSize(statistics.S.CurrentSize-sizeRemoved)` ?

Comment (simonqbs) [inline server/monitoring/monitoring.go]: I dont' think it matters? Just tested and got same results.
Anyway, the stats could probably be moved inside database.DB.Update{} where a transaction is already started to make monitoring run a bit faster (tried this and it does run a bit faster).

Comment (Depado) [inline server/monitoring/monitoring.go]: Well if I see correctly, you're setting the S.CurrentSize to the size you deleted, and that's not relevent I think. 

Yes you're right about the transaction ^^ My code is really far from being perfect, I appreciate all the help I can get to improve myself :)

**Comment by Depado** [comment]
Merging this anyway, that's close to what I would do anyway.

**Pull request merged**